src/pages/RestrictedAction/Workspace/WorkspaceAdminRestrictedAction.tsx

```diff
@@ -39,7 +38,7 @@ function WorkspaceAdminRestrictedAction({policyID}: WorkspaceAdminRestrictedActi
         >
             <HeaderWithBackButton
                 title={translate('workspace.restrictedAction.restricted')}
-                onBackButtonPress={Navigation.goBack}
+                onBackButtonPress={Navigation.resetToHome}
```

Comment:
Same reasoning. we can revert back to `Navigation.goBack`

Reply:
Adjusted